pio/lib/Globals/Globals.h

```diff
@@ -110,11 +110,16 @@ extern Ticker flasher;
 #define RTCVALIDFLAG 0xCAFEBABE
 
 // sleep management
-#define RTCSLEEPADDR 5
-#define MAXSLEEPTIME 3600UL //TODO
+#define RTCSLEEPADDR 5      //And +1 for the valid flag
+#define MAXSLEEPTIME 30UL   //Check the accelerometer every 30 secs for a command   //3600UL //TODO
```

Comment:
this point makes it already clear that this idea reduces the battery lifetime by a factor of ~10x!
No matter the optimization that is a significant issue that make this idea hard to happen